crates/credentials/src/pex/v2/presentation_definition.rs

```diff
@@ -0,0 +1,230 @@
+use jsonschema::{Draft, JSONSchema};
+use serde::{Deserialize, Serialize};
+use serde_json::Value as JsonValue;
+use serde_with::skip_serializing_none;
+use std::collections::HashMap;
+
+/// Presentation Exchange
+///
+/// Presentation Exchange specification codifies a Presentation Definition data format Verifiers
+/// can use to articulate proof requirements, and a Presentation Submission data format Holders can
+/// use to describe proofs submitted in accordance with them.
+///
+/// See [Presentation Definition](https://identity.foundation/presentation-exchange/#presentation-definition)
+/// for more information.
+#[skip_serializing_none]
```

Comment:
So what's the intent of [`skip_serialization_none`](https://docs.rs/serde_with/latest/serde_with/attr.skip_serializing_none.html)? 

Is it such that serialization/deserialization doesn't throw exceptions in the event of optional fields missing?

Reply:
I believe it makes it so that when serializing a struct to JSON, fields who's value is set to `Option::None` will be skipped. If this isn't set, then the fields woulb be serialized as `null`.

A more explicit example below. 

Without the `skip_serializing_none` macro
```json
{
  "foo": 1
  "bar": null
}
```


With the macro

```json
{
  "foo": 1
}
```

Reply:
Yup that makes sense, thanks!